internal/exec/worflow_utils.go

```diff
@@ -33,8 +33,12 @@ func executeWorkflowSteps(workflowDefinition c.WorkflowDefinition) error {
 			var stepStack = strings.TrimSpace(step.Stack)
 			var finalStack = ""
 
-			// The workflow `stack` attribute overrides any stack in the `command` (if specified)
-			// The step `stack` attribute overrides any stack in the `command` (if specified) and the workflow `stack` attribute
+			// The stack defined on the command line (`atmos workflow <name> -f <file> -s <stack>`) has the lowest priority
+			// The workflow `stack` attribute overrides the stack in the `command` (if specified) or on the command line
+			// The step `stack` attribute overrides the stack in the `command`, the workflow `stack` attribute, and on the command line
```

Comment:
This is not the normal order of precedence for a Linux command: almost always (I cannot think of an exception), the command line settings override config file settings. I haven't thought about whether or not this use case warrants the exception, but if it does and you really want the command line option to have the lowest precedence, this needs to be prominently documented, including in the command-line help text.

Reply:
we can create generic workflows to e.g. plan a set of components.
we don't specify stacks for it, we specify a stack on the command line.
but for each step, we are able to override the stack.
that's why I made the command line the lowest priority.

@Nuru @osterman

Reply:
the thinking is, if we override the stack on the command line, then whatever stack is specified in the workflow steps, it's not usable at all (and useless).

this situation should never happen in reality b/c you either create a workflow to deploy a set of components into the same stack (for which you can specify the stack in the workflow itself or on the command line), or you create a workflow to deploy a single component into many stacks (which are provided in the workflow steps).
there are the two main ways to create workflows.

in any case, changing priorities is just changing a few lines of code - we need to discuss and come up with the final solution.

Reply:
workflow stacks on the command line will be addressed in a separate PR after more discussion